app-server/app/src/main/scala/scene/Scene.scala

```diff
@@ -0,0 +1,100 @@
+package com.azavea.rf.scene
```

Comment:
I think we should try and keep database interaction logic outside of what would interact with the routes as much as possible to prevent this file from getting excessively large like the `User.scala` model.